docs/features/tuples.md

```diff
@@ -230,3 +230,65 @@ M1((1, ()=>(2, 3)));         // the first overload is used due to "exact match"
 
 ``` 
 
+Language grammar changes
```

Comment:
Ultimately I think you want to integrate the related sections of the spec. That is, for each context integrate the syntax, semantics, and dynamic semantics, rather than having the prose above and syntax below without relating them to each other.